components/ui/InfiniteCanvas.tsx

```
@@ -1780,85 +1780,99 @@ export default function InfiniteCanvas() {
+                                <input
+                                    type="color"
+                                    onChange={(e) => {
+                                        const newColor = e.target.value;
+                                        if (
+                                            moveMode &&
+                                            selectedId &&
+                                            selectedShape
+                                        ) {
+                                            switch (selectedShape) {
+                                                case 'line':
+                                                    const newLines = [...lines];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)